tools/python/upload_and_run_browserstack_tests.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import json
+import time
+
+import requests
+
+parser = argparse.ArgumentParser("Upload and run BrowserStack tests")
+
+parser.add_argument("--id", type=str, help="BrowserStack user ID")
+parser.add_argument("--token", type=str, help="BrowserStack token")
+parser.add_argument("--test_platform", type=str, help="Testing platform, i.e., XCUITest or Espresso")
```

Comment:
if there are two valid options, can we just constrain them here to catch typos earlier and with an obvious error message?
e.g., `choices=['espresso', ...]`